app/src/main/java/com/pwhs/quickmem/presentation/app/home/recent/study_set/AllRecentAccessStudySetsUiAction.kt

```
@@ -0,0 +1,5 @@
+package com.pwhs.quickmem.presentation.app.home.recent.study_set
+
+sealed class AllRecentAccessStudySetsUiAction() {
+
+}
```

detekt (1.23.8)

[warning] The class or object AllRecentAccessStudySetsUiAction is empty.

(detekt.empty-blocks.EmptyClassBlock)